cert/init.sls

```diff
@@ -43,4 +43,10 @@ cert_packages:
     - mode: {{ key_mode }}  
   {% endif %}
 
+{% if grains['os_family']=="Debian" %}
+  cmd.run:
+    - name: update-ca-certificates
+    - runas: root
```

Comment:
I think this will benefit with
```
  - onchanges:
    - file: {{ cert_dir }}/{{ name }}
```
that way if the files are already set it won't run update-ca-certificates every time.